src/components/Navbar.js

```diff
@@ -9,13 +11,21 @@ const navLinks = [
 function Navbar() {
   return (
     <>
-      {
+      <div className="navbar-container">
```

Comment:
- [ ] The bottom border should be darker as seen in the original design. 
### Your design
![image](https://github.com/manzitresor/book_store/assets/45460192/2ea29f6b-596e-4de7-a6c4-302e78a5a2ce)
### Required Design
![image](https://github.com/manzitresor/book_store/assets/45460192/ee1ddaf7-aa56-4918-9109-09719ba48f51)